### PR TITLE
Fix search input margin

### PR DESCRIPTION
**Pull request by rionner**
## Proposed changes

- empty feeds had narrow sizing
- search results view of search bar had margin to the right

## Types of changes

_What types of changes does your code introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Unit Tests (only unit tests were added)
- [ ] Sprint Deployment

## How has this been tested?

- manual UI testing

## Checklist

- [x] I have added the current sprint milestone to this PR.
- [x] I have attached this PR to all relevant cards.
- [x] Any dependent changes have been merged and released if needed.
- [x] My code follows the code style of this project.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have added tests (unit and/or acceptance) that prove my fix is effective or that my feature works.
- [ ] My changes have manual deployment steps, and I have documented these in the current sprint's deployment card.
- [ ] I have checked off and verified that all items within the “Definition of Done” section on this work item have been met.

## Potential opportunities or challenges uncovered as a result of this change

- [ ] I have created inbox cards for any opportunities or future challenges that I was able to recognize in the process making this change.

### CODE REVIEW

**Comment by rionner** [comment]
[Filter bar in Communities does not remain static, moves when searching](https://trello.com/c/G6txuPLD/1926-filter-bar-in-communities-does-not-remain-static-moves-when-searching)

**Comment by rionner** [comment]
[Filter bar in Communities does not remain static, moves when searching](https://trello.com/c/G6txuPLD/1926-filter-bar-in-communities-does-not-remain-static-moves-when-searching)

**Comment by rionner** [comment]
[Filter bar in Communities does not remain static, moves when searching](https://trello.com/c/G6txuPLD/1926-filter-bar-in-communities-does-not-remain-static-moves-when-searching)

File: assets/scss/atoms/_custom-search-clear-input.scss [hunk 1]
```diff
@@ -10,7 +12,10 @@
         height: $x-large;
     }
     .transparent-button:first-of-type { left: $x-large; }
-    .transparent-button:last-of-type { right: $x-large; }
+    .transparent-button:last-of-type {
+        right: $x-large;
+        margin-right: rem-calc(-45);
```
Comment (peteclodi) [inline assets/scss/atoms/_custom-search-clear-input.scss]: @rionner Is it possible to use `-$x-large` instead of having to hardcode the `rem-calc(45)`?

**Comment by rionner** [comment]
@peteclodi Updated! I'm kind of surprised that `-$x-large` worked, but it does!

**Comment by peteclodi** [comment]
@rionner It is kindof a weird language quirk. I've used it in other languages, so I thought it might work in JS and save us some issues of having to write out the opposite of our predefined size variables.

**Pull request merged**